src/SourceBuild/tarball/content/patches/vstest/0001-Condition-win7-x86-RID.patch

```diff
@@ -0,0 +1,26 @@
+From 0fc9ffcb249411f9911a6e13f939eaf95bf219c8 Mon Sep 17 00:00:00 2001
```

Comment:
Integrating this patch into the vstest repo is handled with https://github.com/microsoft/vstest/pull/2981.